src/library/cmd_reader.h

```diff
@@ -2,6 +2,8 @@
 
 #include "impl/base.h"
 
+using namespace std::experimental::filesystem;
```

Comment:
Using directives in headers are not good design because it affects everyone including your header and can introduce namespace collisions.